src/main/scala/org/broadinstitute/dsde/workbench/avram/config/AvramConfig.scala

```diff
@@ -0,0 +1,5 @@
+package org.broadinstitute.dsde.workbench.avram.config
+
+case class AvramConfig(serviceVersion: String,
+                        clientId: String,
+                        googleProject: String)
```

Comment:
Are these just examples or do you envision using `clientId` and `googleProject` in the future? (and if so, how?)

Reply:
they're examples for now - making sure we can pull secrets and env. variables!